tasks/install_binary.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: Get fabio checksums file
```

Comment:
I didn't get it. Why?
Yes, uri module with return_content will get me a page with checksums of binaries for all platforms.
Still somehow I'll need to extract checksum for only one for fabio-1.3.4-go1.7.3-linux_amd64

FYI
>The dependency on httplib2 was removed in Ansible 2.1